LICENSE

```diff
@@ -0,0 +1,24 @@
+The MIT License
+
+MIT License
+
+Copyright (c) 2021 GuillotinaWeb
```

Comment:
From a legal point of view I don't know if this is valid. For now I'd say the code is copyrighted to Jordi Collell and maybe in a future we can transfer the copyright to Plone Foundation if we want

Reply:
It probably needs to be adapted. This LICENSE is just as Github generates, I have only changed year and the name (where perhaps it is better to put Jordi Collell instead of GuillotinaWeb).

Reply:
And the same for CONTRIBUTING.md, it's copy-pasted from Preact.